.github/workflows/tests.yml

```diff
@@ -8,7 +8,12 @@ jobs:
       # We support Node Latest, Active LTS, and the most recent Maintenance LTS.
       # See https://nodejs.org/en/about/releases/ for Node release schedule.
       matrix:
-        version: [14, 16, 18]
+        version: [
```

Comment:
I kinda prefer "node" as the key, because "version" is more ambiguous ("os" is also a kind of version)

Reply:
Yes, that is totally true. Done!